app.py

```diff
@@ -106,6 +106,20 @@ def solver(os_name: str, os_version: str, python_version: str) -> None:
     graph = GraphDatabase()
     graph.connect()
     result = graph.purge_solver_documents(os_name=os_name, os_version=os_version, python_version=python_version)
+    if result == 0:
+        _LOGGER.warning(
+            f"No solver entries to purge were found for"
+            f"--os-name: {os_name},"
+            f"--os-version: {os_version},"
+            f"--python-version: {python_version}."
+        )
+        if graph.solved_software_environment_exists(
+            os_name=os_name, os_version=os_version, python_version=python_version
+        ):
+            _LOGGER.info(
```

Comment:
I think we can treat this as an error (so we get notifications in monitoring systems):

```suggestion
            _LOGGER.error(
```